src/components/MultiHopTrade/components/TradeInput/TradeInput.tsx

```diff
@@ -216,6 +217,8 @@ export const TradeInput = ({ isCompact, tradeInputRef, onChangeTab }: TradeInput
       if (!tradeQuoteStep) throw Error('missing tradeQuoteStep')
       if (!activeQuote) throw Error('missing activeQuote')
 
+      if (!isExecutableTradeQuote(activeQuote)) throw new Error('Cannot submit a trade rate')
```

Comment:
same generic `Unable to execute trade` error?

Reply:
66415441ab